app/src/main/res/layout-v21/seekbar_pref.xml

```diff
@@ -51,6 +51,28 @@
         />
   </LinearLayout>
 
+  <LinearLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content">
+
+    <TextView
+        android:id="@+id/pref_preview"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="@string/prefs_preview"
+        android:gravity="center"
+        android:layout_weight="1"/>
+
+    <TextView
+        android:id="@+id/arabic_preview"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="معاينة"
```

Comment:
should be `@string/prefs_preview` and we should add to `values-ar/strings.xml`

Reply:
oh - i see why now - will add another one just for this and make it non-translatable